### PR TITLE
feat: Added 4.0 GA stable banner to api explorer

### PR DESCRIPTION
**Pull request by jeremytchang**
- Shows banner containing information on API 4.0 GA and a link to
dev portal documentation
- Displays close button that hides the banner and sets localstorage
to not show in the future
- Banner is only shown if it has not been hidden before and the
instance's 4.0 spec is not stable or current.

Screen Shot 2022-01-26 at 5 27 03 PM
Screen Shot 2022-01-26 at 5 26 45 PM

Copy in screenshot slightly out of date. Now says:
`API 4.0 moves from Beta to General Availability in Looker 22.4 with additions and possible breaking changes. Read the announcement to see how this affects you.`

### CODE REVIEW

**Comment by jeremytchang** [comment]
Will pull in Sharon for PM copy approval.

**Comment by jeremytchang** [comment]
Sharon has approved of the message.

**Comment by jeremytchang** [comment]
@josephaxisa I need guidance on being able to test locally. It's not viable iterating on tests by pushing up commits and triggering CI.

**Comment by jkaster** [comment]
This should be a `feat:` not a `fix:`

**Comment by jkaster** [comment]
From the root of the repo, `yarn test:apix` followed by `yarn test:apix-e2e` if you want to be really thorough.

`yarn run` lists all commands

File: packages/api-explorer/src/components/Banner/Banner.tsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+            href="https://developers.looker.com/api/advanced-usage/version-4-ga"
+            target="_blank"
+          >
+            <ButtonOutline size="small">Announcement</ButtonOutline>
+          </Link>
+        }
+      >
+        API 4.0 is generally available in Looker 22.4 and transitions from Beta
+        to Stable with additive and breaking changes. Read the announcement to
+        see how this affects you!
```
Comment (jkaster) [inline packages/api-explorer/src/components/Banner/Banner.tsx:60]: This message is short and clear, but I'm concerned about the "breaking changes" statement for moving from beta to GA. Maybe this works. Will have to get docs/PM feedback.

```suggestion
        API 4.0 is generally available in Looker 22.4 and transitions from Beta
        to Stable with additions and changes. Read the announcement to
        see how this affects you!
```

**Comment by jeremytchang** [comment]
@jkaster Chris Michel is fine with messageBar component. Updated with messageBar component and (should be) passing unit tests.

**Review by jkaster — CHANGES_REQUESTED** [review]
Just working on text

File: packages/api-explorer/src/components/Banner/Banner.tsx (hunk 1 above)
Comment (jeremytchang) [reply packages/api-explorer/src/components/Banner/Banner.tsx:60]: Sharon had approved, but of original messaging but will ping her again.

Comment (jkaster) [reply packages/api-explorer/src/components/Banner/Banner.tsx:60]: I've updated to Sharon's latest text

**Comment by jeremytchang** [comment]
Chris Michel designer approves.

**Review by jkaster — APPROVED** [review]
Looks good.

**Pull request merged**